plugins/inputs/vsphere/README.md

```diff
@@ -118,52 +128,73 @@ vm_metric_exclude = [ "*" ]
     "storageAdapter.write.average",
     "sys.uptime.latest",
   ]
-  # host_metric_exclude = [] ## Nothing excluded by default
-  # host_instances = true ## true by default
-
-  ## Clusters
-  # cluster_include = [ "/*/host/**"] # Inventory path to clusters to collect (by default all are collected)
-  # cluster_metric_include = [] ## if omitted or empty, all metrics are collected
-  # cluster_metric_exclude = [] ## Nothing excluded by default
-  # cluster_instances = false ## false by default
-
-  ## Datastores
-  # cluster_include = [ "/*/datastore/**"] # Inventory path to datastores to collect (by default all are collected)
-  # datastore_metric_include = [] ## if omitted or empty, all metrics are collected
-  # datastore_metric_exclude = [] ## Nothing excluded by default
-  # datastore_instances = false ## false by default
-
-  ## Datacenters
-  # datacenter_include = [ "/*/host/**"] # Inventory path to clusters to collect (by default all are collected)
-  datacenter_metric_include = [] ## if omitted or empty, all metrics are collected
-  datacenter_metric_exclude = [ "*" ] ## Datacenters are not collected by default.
-  # datacenter_instances = false ## false by default
-
-  ## Plugin Settings
-  ## separator character to use for measurement and field names (default: "_")
+  host_metric_exclude = []
+
+  ## If true, gather host instance metrics.
+  # host_instances = true
+
+  ## Inventory paths for the clusters to gather metrics on.  If empty,
+  ## all clusters are included.
+  ##   ex: cluster_include = ["/*/cluster/**"]
```

Comment:
Make this path "/**" instead. Gets rid of that permission issue someone ran into a while ago. Requires a change to the default struct initializer in vsphere.go. See #5613